analytic_engine/src/table/mod.rs

```diff
@@ -219,7 +219,7 @@ impl<
             .await
             .map_err(|e| Box::new(e) as _)
             .context(AlterSchema { table: self.name() })?;
-        Ok(1)
+        Ok(0)
```

Comment:
If those alter statements always return 0 when success, isn't it better to just return `Result<()>`, and let the upper layer to return expected `affected_row`?

IMO, `affected_row` have no meaning in `Table` layer, only useful in upper layer.

Reply:
yes! I'll change it later

Reply:
@dust1 Don't hurry, Let wait and see what others view is.

Reply:
We have some (intend to be informal) discussions in [slack]( https://ceresdbcommunity.slack.com/archives/C03KG4E4J3Z/p1657251293081759), if this link doesn't work you can try to join with [this one](https://join.slack.com/t/ceresdbcommunity/shared_invite/zt-1au1ihbdy-5huC9J9s2462yBMIWmerTw).

And the question we are discussing is how to define `affected_rows`, should it be "engine independent" or not.

Reply:
Considering CeresDB is designed to be multi-engine, we'd better not have hypotheses about how an engine will implement an operation. In this case `analytic_engine` won't change any row in an `ALTER TABLE`, but maybe an engine in the future would. So the `Table` trait should keep the return value.